tests/filecheck/backend/x86/convert_func_to_x86.mlir

```diff
@@ -1,5 +1,19 @@
 // RUN: xdsl-opt -p convert-func-to-x86-func --split-input-file  %s | filecheck %s
 
+// CHECK:       builtin.module {
+
+func.func @foo_const() -> i32 {
+  %0 = "test.op"(): () -> i32
+  func.return %0: i32
+}
+// CHECK-NEXT:    x86_func.func @foo_const(%0 : !x86.reg<rsp>) -> !x86.reg<rax> {
+// CHECK-NEXT:      %1 = "test.op"() : () -> i32
+// CHECK-NEXT:      %2 = builtin.unrealized_conversion_cast %1 : i32 to !x86.reg
+// CHECK-NEXT:      %3 = x86.get_register : () -> !x86.reg<rax>
+// CHECK-NEXT:      %4 = x86.rr.mov %2, %3 : (!x86.reg, !x86.reg<rax>) -> !x86.reg<rax>
+// CHECK-NEXT:      x86_func.ret
+// CHECK-NEXT:    }
+
```

Comment:
This is missing the test case separator (for `--split-input-file`) `// -----`.
I think it'd be could if these tests are separate.